src/mn_setup.c

```diff
@@ -1881,6 +1881,13 @@ static setup_menu_t stat_settings2[] = {
     {"Level Stats Format", S_CHOICE, H_X, M_SPC, {"hud_stats_format"},
      .strings_id = str_stats_format},
```

Comment:
I guess this goes under "Widget Appearance" like before.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1992/commits/281d52274a9483415ecf3fe7f6786e314f829338